packages/eslint-plugin-react-native/README.md

```diff
@@ -0,0 +1,39 @@
+# @getluko/eslint-plugin-react-native
+
+This library was generated with [Nx](https://nx.dev).
+
+## Building
+
+Run `nx build eslint-plugin-react-native` to build the library.
+
+## Running unit tests
+
+Run `nx test eslint-plugin-react-native` to execute the unit tests via [Jest](https://jestjs.io).
+
+# @getluko/eslint-plugin-mobile-guidelines
+
+## Installation
+
+```
+yarn add @getluko/eslint-plugin-mobile-guidelines --dev
+```
+
+## Configuration
+
+`.eslintrc.js`
+
+```
+module.exports = {
+  plugins: ['@getluko/mobile-guidelines'],
+  rules: {
+    '@getluko/mobile-guidelines/lower-dash-case-test-id': 1,
+  }
+```
+
+## Rules
+
+### [@getluko/mobile-guidelines/lower-dash-case-test-id](src/docs/rules/lower-dash-case-test-id.md)
```

Comment:
Should we keep the documentation at the root for now? We will not have 100 to maintain so it should be easier

Reply:
I added a preview also at the root README. But I think it's cleaner to keep the rule markdown file as well to well explain how configure the rule.